resources/views/individuals/partials/progress.blade.php

```diff
@@ -31,6 +38,13 @@
                 <p id="cannot-publish-explanation">
                     {{ __('You must attend an orientation session and fill in all the required information before you can publish your page.') }}
                 </p>
+                @can('publish', $individual)
+                    <x-interpretation class="interpretation--start" name="{{ __('Publish page', [], 'en') }}"
+                        namespace="publish_page" />
+                @else
+                    <x-interpretation class="interpretation--start" name="{{ __('Publish page', [], 'en') }}"
+                        namespace="publish_page-disabled" />
+                @endcan
```

Comment:
This still needs to be updated. The publish button is inside of the `@cannot` from line 37. This publish interpretation should be moved up to after line 33. Then you can also remove the `@can` from here.